salt/cloud/clouds/vmware.py

```diff
@@ -3294,6 +3294,28 @@ def create_snapshot(name, kwargs=None, call=None):
                                                             vm_ref.snapshot.currentSnapshot)}
 
 
+def _find_snapshot_by_name(vm_ref, name):
+    def find_snap(tree, name):
```

Comment:
Can you rename ``tree`` to ``base_snapshot`` since tree doesn't really mean anything in the context.